src/window.rs

```diff
@@ -1115,6 +1115,43 @@ impl Window {
     self.window.set_progress_bar(_progress)
   }
 
+  /// Sets the badge count on the taskbar
+  ///
+  /// ## Implementation Details
+  /// - _desktop_filename is for linux ONLY!
+  /// 
+  /// ## Platform-specific
+  ///
+  /// - **Windows**: Windows allows to set an arbitrary .ico file. See set_overlay_icon
+  /// - **Linux / macOS**: Badge count is app-wide and not specific to this window. Only supported desktop environments with `libunity` (e.g. GNOME).
+  /// - **iOS / Android:** Unsupported.
+  #[inline]
+  pub fn set_badge_count(&self, _count: Option<i64>, _desktop_filename: Option<String>) {
+    #[cfg(target_os = "macos")]
+    self.window.set_badge_count(_count);
+
+    #[cfg(any(
+      target_os = "linux",
+      target_os = "dragonfly",
+      target_os = "freebsd",
+      target_os = "netbsd",
+      target_os = "openbsd",
+    ))]
+    self.window.set_badge_count(_count, _desktop_filename);
+  }
+
+  /// Sets the Overlay Icon **(Windows only)**
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Linux / macOS**: See set_taskbar_badge
+  /// - **iOS / Android:** Unsupported.
+  #[inline]
+  pub fn set_overlay_icon(&self, _icon_path: Option<String>) {
```

Comment:
I think this should use the `Icon` type instead of it being a path.
```suggestion
  pub fn set_overlay_icon(&self, icon: Option<Icon>) {
```

Reply:
also since this Windows only API, it should be moved to `WindowExtWindows` trait